src/utils/exportUtils/Exporter.ts

```diff
@@ -77,7 +77,7 @@ export default abstract class Exporter {
 
     protected async downloadZIP(): Promise<string | void> {
         const brand = SdkConfig.get().brand;
-        const filenameWithoutExt = `${brand} - Chat Export - ${formatFullDateNoDay(new Date())}`;
+        const filenameWithoutExt = `${brand} - ${this.room.name} - Chat Export - ${formatFullDateNoDay(new Date())}`;
```

Comment:
We probably need to sanitize the name to not include vars which are typically invalid - https://www.mtu.edu/umc/services/websites/writing/characters-avoid/

Reply:
Its not included in the name of the file.${variable} resolves to a value hence the character itself will not be included as it is concatenation in string with template literals .this is what  you are saying or Are you talking about the  need to remove whitespaces with "_" or "-"?

Reply:
Ok now I got it ..Thanks